orchestra/tests/test_todos.py

```diff
@@ -602,3 +632,241 @@ def test_conditional_skip_remove_todos_from_template(self):
+                    {
+                        'id': 2,
+                        'description': 'todo parent 2',
+                        'items': [
+                            {
+                                'id': 22,
+                                'description': 'todo child 2-2',
+                                'items': []
+                            },
+                            {
+                                'id': 21,
+                                'description': 'todo child 2-1',
+                                'items': [
+                                    {
+                                        'id': 211,
+                                        'description': 'todo child 2-1-1',
+                                        'items': []
+                                    }
+                                ]
+                            }
+                        ],
+                        'remove_if': [{
+                            'prop': {
+                                'operator': '==',
+                                'value': True
+                            }
+                        }]
+                    },
+                    {
+                        'id': 1,
+                        'description': 'todo parent 1',
+                        'items': [{
+                            'id': 11,
+                            'description': 'todo child 1-1',
+                            'items': [],
+                            'skip_if': [{
+                                'prop': {
+                                    'operator': '==',
+                                    'value': True
+                                }
+                            }]
+                        }]
+                    }]},
+        )
+        self.empty_todo_list_template = TodoListTemplateFactory(
+            slug='empty-template-slug',
+            name='Empty template name',
+            description='Empty template description',
+            conditional_property_function={
+                'path': 'orchestra.tests.test_todos'
+                        '._get_test_conditional_props'
+            },
+            todos={}
+        )
+
+        self.import_url_name = (
+            'orchestra:todos:import_todo_list_template_from_spreadsheet')
+        self.template_admin_name = 'admin:orchestra_todolisttemplate_change'
+        # Object actions don't appear to have a `reverse` pattern, so
+        # we explicitly spell out the URL.
+        self.export_url_pattern = (
+            '/vip/orchestra/todolisttemplate/{}/actions/export_spreadsheet/')
+
+    @patch('orchestra.todos.import_export._upload_csv_to_google')
+    def test_export_spreadsheet(self, mock_upload):
+        mock_upload.return_value = 'https://redirect.com/the_spreadsheet'
+        response = self.request_client.get(
+            self.export_url_pattern.format(self.full_todo_list_template.id))
+        self.assertEqual(response.status_code, 302)
```

Comment:
This fails with a 404 in CI but succeeds in dev. I suspect somehow `django-object-actions` isn't being registered/installed properly, despite being installed in the `Install dependencies` step. I'll keep poking around, but any ideas are welcome. The other tests pass in both environments.

Reply:
The proper `reverse` fixed it!